src/TrueLayer/Mandates/MandatesApi.cs

```diff
@@ -52,5 +54,48 @@ public async Task<ApiResponse<CreateMandateResponse>> CreateMandate(CreateMandat
                 cancellationToken
             );
         }
+
+        /// <inheritdoc />
+        public async Task<ApiResponse<MandateDetailUnion>> GetMandate(string mandateId, MandateType mandateType, CancellationToken cancellationToken = default)
+        {
+            mandateId.NotNullOrWhiteSpace(nameof(mandateId));
+
+            ApiResponse<GetAuthTokenResponse> authResponse = await _auth.GetAuthToken(new GetAuthTokenRequest($"recurring_payments:{mandateType}"), cancellationToken);
+
+            if (!authResponse.IsSuccessful)
+            {
+                return new(authResponse.StatusCode, authResponse.TraceId);
+            }
+
+            return await _apiClient.GetAsync<MandateDetailUnion>(
+                new Uri(_baseUri, $"/v3/mandates/{mandateId}"),
```

Comment:
I think the resulting URI will be incorrect, it should be just `new Uri(_baseUri, "mandateId)`.


Having a look at how the [GetPayment method](https://github.com/1ststop/truelayer-dotnet/blob/dfc49f824b29d0bceb80c4fec43977ad70a541ab/src/TrueLayer/Payments/PaymentsApi.cs#L88) works might help here ;)